src/node/ledger_secrets.h

```diff
@@ -111,6 +111,16 @@ namespace ccf
       ledger_secrets.emplace(initial_version, make_ledger_secret());
     }
 
+    void init_from_map(LedgerSecretsMap&& ledger_secrets_)
```

Comment:
Could you remove the `LedgerSecrets` constructor that takes a `LedgerSecretsMap&&`?